frameworks/java/prewarm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "loading" > /workspace/prewarm.status
+
+# prewarm by starting the gradle daemon. Running an initial test build will also speed things up a bit
+cd /runner/frameworks/java && gradle --daemon --offline test
```

Comment:
Using `top` on a local container, it looks like an extra 256mb of RAM is taken up once I start the Gradle daemon.